internal/syncer/connector/bitbucket.go

```
@@ -0,0 +1,1053 @@
+	}
+	var page map[string]any
+	_, err := c.getJSON(ctx, c.apiURL("/repositories/"+url.PathEscape(c.workspace), url.Values{
+		"pagelen": {"1"},
+		"fields":  {"pagelen"},
+	}), &page)
+	if err != nil {
+		var httpErr *bitbucketHTTPError
+		if errors.As(err, &httpErr) {
+			switch httpErr.Status {
+			case http.StatusUnauthorized:
+				return fmt.Errorf("Invalid or expired Bitbucket credentials (HTTP 401).")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not end with punctuation or newlines

(staticcheck)